docs/content/en/docs/tutorials/developer-journey.md

```diff
@@ -0,0 +1,159 @@
+---
+title: "Developer Journey with Buildpacks"
+linkTitle: "Developer Journey"
+weight: 100
+---
+
+# Skaffold Developer Journey with Buildpacks Tutorial
+
+## Introduction
+
+### What is this project?
+
+Skaffold allows developers to easily transition from local development on minikube to remote development on an enterprise Kubernetes cluster managed by IT. During the transition from local to remote deployment, a security team might ask a developer to patch a library with a specific vulnerability in it. This is where Skaffold's support for buildpacks comes in handy. In this tutorial, you'll start out deploying an application locally, swap out buildpacks in the `skaffold.yaml` file to use the latest libraries, and then deploy the application to a remote Kubernetes cluster.
+
+For a guided Cloud Shell tutorial on how a developer's journey might look in adopting Skaffold, follow:
+
+[![Open in Cloud Shell](https://gstatic.com/cloudssh/images/open-btn.svg)](https://ssh.cloud.google.com/cloudshell/editor?cloudshell_git_repo=https://github.com/GoogleContainerTools/skaffold/docs/content/en/docs/tutorials&cloudshell_workspace=.&cloudshell_tutorial=developer-journey.md)
+
+### What you'll learn
+
+* How to develop on minikube locally and move to an enterprise managed kubernetes cluster with ease
+* How to easily switch buildpack versions to comply with security demands
+
+___
+
+**Time to complete**: <walkthrough-tutorial-duration duration=15></walkthrough-tutorial-duration>
+Click the **Start** button to move to the next step.
+
+## First steps
+
+### Prepare the Environment
+Create a project called `skaffold-tutorial` with a valid billing account in the Google Cloud [Manage Resources Console](https://console.cloud.google.com/cloud-resource-manager)
+
+Run the start script to deploy a GKE cluster we will use later and install the latest version of Skaffold. Although Skaffold is already installed on cloud shell, we'll install the latest version for good measure.
+```bash
+chmod +x start.sh && ./start.sh
+```
+
+### Start a Minikube cluster
+
+We'll use `minikube` as our local kubernetes cluster of choice.
+
+Run:
+```bash
+minikube start
+```
+
+## Run the App to Minikube Using Skaffold
+
+### Deploy the App
+Run:
+```bash
+skaffold dev
+```
+
+**Important:** note the software versions under the `DETECTING` phase of the buildpack output. These will be important later.
+
+Open a second terminal. In order to connect to the application to make sure its working, you need to start the minikube load balancer by running:
+```bash
+minikube tunnel
```

Comment:
You can add port-forwarding in the Skaffold config to do this automatically:
https://skaffold.dev/docs/pipeline-stages/port-forwarding/

Reply:
Thats tight! I'll test that real quick.

Reply:
Ok I think that `--port-forward` will just forward to a NodePort service type. I am using a LoadBalancer service type so that when the app is deployed in the cloud there's a global IP that can be easily reached. `minikube tunnel` appears to start a service that will create a Load Balancer properly.